bin/run-tests-in-docker.sh

```diff
@@ -26,6 +26,7 @@ docker run \
     --mount type=bind,src="${PWD}/tests",dst=/opt/test-runner/tests \
     --mount type=tmpfs,dst=/tmp \
     --volume "${PWD}/bin/run-tests.sh:/opt/test-runner/bin/run-tests.sh" \
+    --volume "${PWD}/bin/run.sh:/opt/test-runner/bin/run.sh" \
```

Comment:
But this file is already in the docker image I think:

https://github.com/exercism/php-test-runner/blob/345554475c862524ee96150d54bc66dd85e3151e/Dockerfile#L34-L35

Reply:
Yes, it is. This enables development changes to the file without re-building the image every time. I found it very slow to rebuild the image and commented that out for faster cycles. Also I experienced problems with small changes not making it into the container (Docker used the cache instead of the changed file for whatever reason).

It can be removed safely, if you don't want it in there.

Reply:
>  I found it very slow to rebuild the image and commented that out for faster cycles. Also I experienced problems with small changes not making it into the container (Docker used the cache instead of the changed file for whatever reason).

These are weird issues that maybe should be solved first. Do you have some examples?

One thing that you should look into is the [`.dockerignore` file](https://github.com/exercism/php-test-runner/blob/main/.dockerignore), which in large part determines how caching works. E.g. I see that the `test` folder is excluded, but it is actually called `tests`. I also see that the `bin/run-in-docker.sh`, `bin/run-tests-in-docker.sh` and `bin/run-tests.sh` files aren't ignored. Maybe you can try updating the `.dockerignore` file first to see if that helps (it should).

Reply:
I looked into the `.dockerignore` (and which files should really be in the production image), and decided not to clutter this PR with those unrelated changes (the is / should be about upgrades). `test` can be removed, but the `tests` must be included or added as a volume. There is some weirdness about PHPUnit sources required for `junit-handler` and loading the PHPUnit PHAR. And some more.

The Docker cache issues I also experience with other projects I run locally. I think it is more about the IDE being containerized and the shell running Docker commands isn't. Whatever really happens, volumes help.

I'll remove the volume, so the PR really is only about the upgrading. That sounds best to me now.